docs/sources/plugins/_index.md

```diff
@@ -4,5 +4,48 @@ type = "docs"
 [menu.docs]
 name = "Plugins"
 identifier = "plugins"
-weight = 8
+weight = 1
 +++
+
+# Plugins
+
+In addition to the wide range of visualizations and data sources that are available immediately after you install Grafana, you can extend your Grafana experience through _plugins_.
+
+You can [install]({{< relref "./installation.md" >}}) one of the plugins built by the Grafana community, or [build one yourself]({{< relref "./developing/development.md" >}}).
+
+Grafana supports three types of plugins: _panels_, _data sources_, and _apps_.
+
+## Panel plugins
+
+Add new visualizations to your dashboard through panel plugins, such as [world maps](https://grafana.com/grafana/plugins/grafana-worldmap-panel), [clocks](https://grafana.com/grafana/plugins/grafana-clock-panel), and [pie charts](https://grafana.com/grafana/plugins/grafana-piechart-panel).
+
+Use panel plugins when you want to:
+
+- Visualize data returned by data source queries.
+- Navigate between dashboards.
+- Control external systems, such as smart home devices.
+
+## Data source plugins
```

Comment:
This section would be a great place to link to the specific tutorials, perhaps at the end of each section. If you're a developer, this tutorial shows you how to build your own data source plugin.

Reply:
I tried to make this document work for both end-users (using plugins) and developers (building plugins). Maybe instead add a link to a "Build a plugin overview" to the Learn more section?